README.md

```diff
@@ -27,6 +27,20 @@ rosdep install --from-paths src --ignore-src --rosdistro=kinetic -y
 
 Compile the code with `catkin_make`.
 
+### Installation on ros_comm
+You will need `compressed_image_transport`, `camera_info_manager`, and `dynamic_reconfigure` packages.  In `~/ros-catkin-ws`, run:
+
+```
+rosinstall_generator compressed_image_transport --rosdistro kinetic --deps --wet-only --tar > kinetic-compressed_image_transport-wet.rosinstall
+rosinstall_generator camera_info_manager --rosdistro kinetic --deps --wet-only --tar > kinetic-camera_info_manager-wet.rosinstall
+rosinstall_generator dynamic_reconfigure --rosdistro kinetic --deps --wet-only --tar > kinetic-dynamic_reconfigure-wet.rosinstall
+wstool merge -t src kinetic-compressed_image_transport-wet.rosinstall
+wstool merge -t src kinetic-camera_info_manager-wet.rosinstall
+wstool merge -t src kinetic-dynamic_reconfigure-wet.rosinstall
+wstool update -t src
+```
+Then rebuild ROS.  Then you may proceed through installation as instructed above.  For more detailed notes see [this blogpost](http://www.venelinpetkov.com/how-to-install-a-raspberry-camera-node-on-ros-kinetic-raspbian-stretch/).
```

Comment:
Looks like the URL changed: https://venelinpetkov.com/2017/11/19/how-to-install-a-raspberry-camera-node-on-ros-kinetic-raspbian-stretch/